Mathlib/SetTheory/Surreal/Basic.lean

```diff
@@ -386,6 +389,50 @@ theorem nat_toGame : ∀ n : ℕ, toGame n = n :=
   map_natCast' _ one_toGame
 #align surreal.nat_to_game Surreal.nat_toGame
 
```

Comment:
What happened to `upperBound_numeric`?

Reply:
In the review for #10566 we decided to get rid of the definition `upperBound` entirely just keep `PGame.bddAbove_of_small`. I don't think we realized that this file relied on specific properties of the witness used. For now, I've duplicated the proof of `PGame.bddAbove_of_small`, but in the future I think long term this proof should be refactored to take the (possibly non-numeric) upper bound given `PGame.bddAbove_of_small` and then use the fact that there is a `Numeric` game greater than any given game to provide a `Numeric` upper bound. 

(I also wrote this in an earlier comment in this PR https://github.com/leanprover-community/mathlib4/pull/10458#discussion_r1510020804)

Reply:
Oh, are you just saying there needs to be a `#noalign`?

Reply:
It should either be present, or `noalign` along with an explanation of its absence living in the PR description.

Reply:
I added `#noalign`s